R/html_print.R

```diff
@@ -57,6 +57,41 @@ html_print <- function(html, background = "white", viewer = getOption("viewer",
   invisible(index_html)
 }
 
+get_console_viewer <- function() {
+  getOption("console_viewer", getOption("viewer", utils::browseURL))
```

Comment:
Maybe this should warn about the absence of the `"console_viewer"` option and say it's going to use the `"viewer"` option instead ?